sympy/core/tests/test_arit.py

```diff
@@ -1459,11 +1459,12 @@ def test_Pow_as_coeff_mul_doesnt_expand():
     assert exp(x + exp(x + y)) != exp(x + exp(x)*exp(y))
 
 
-def test_issue_3514():
+def test_issue_3514_18626():
     assert sqrt(S.Half) * sqrt(6) == 2 * sqrt(3)/2
     assert S.Half*sqrt(6)*sqrt(2) == sqrt(3)
     assert sqrt(6)/2*sqrt(2) == sqrt(3)
     assert sqrt(6)*sqrt(2)/2 == sqrt(3)
+    assert sqrt(8)**Rational(2, 3) == 2
```

Comment:
IMO, it would be better to test this in a separate function, `test_issue_18626`.

Reply:
https://github.com/sympy/sympy/issues/18626#issuecomment-584773658

Reply:
IMO we should go by the following way, so that we can have a track of the issue resolved with its test
```
# issue 18626
assert sqrt(8)**Rational(2, 3) == 2
```

Reply:
IMO, there might be a reason behind @smichr putting this issue's regression test in that test list.

Reply:
yes, I added the issue number after the existing number in the test function's name since I thought they both are basically similar issues of getting autosimplification to work for rational roots of integers.

Reply:
we do this in many places in SymPy's test suite.

Reply:
> yes, I added the issue number after the existing number in the test function's name since I thought they both are basically similar issues of getting autosimplification to work for rational roots of integers.

Yeah I was thinking exactly the same reason behind why u did that.